README.md

```diff
@@ -18,51 +16,50 @@ A star ⭐️ is welcome.
 [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/dbpunk-labs/db3/badge)](https://www.gitpoap.io/gh/dbpunk-labs/db3)
 [![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/sz3bmZx2uh)
 
-
 **English | [中文](./README_zh_cn.md)**
 
-# What's DB3 Network
+## What's DB3 Network?
 
-DB3 Network is an open-source decentralized firebase firestore alternative to build dapps quickly with minimal engineering effort.
+DB3 Network is an open-source, decentralized firebase firestore alternative to build dApps quickly and less challenging.
```

Comment:
I think the engineering effort learned from tooljet is better. 
>ToolJet is an open-source low-code framework to build and deploy internal tools quickly with minimal engineering effort

Reply:
OK